src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -1061,7 +1079,7 @@ private void onDecodeSubscribe(
                         return;
```

Comment:
Try to avoid multi-return methods, instead break the loop and let the default return at end of method take effect.